src/hooks/settingsHooks/useGetSettings.ts

```diff
@@ -116,12 +117,37 @@ const convertFromBe = (backendSettings: BackendSiteSettings): SiteSettings => {
   }
 }
 
+const extractPassword = (
+  passwordData: BackendPasswordSettings
+): string | null => {
+  const { isAmplifySite } = passwordData
+  if (!isAmplifySite) return null
+
+  return passwordData.password
+}
+
 export const useGetSettings = (
   siteName: string
 ): UseQueryResult<SiteSettings> => {
   return useQuery<SiteSettings>(
     [SETTINGS_CONTENT_KEY, siteName],
-    () => SettingsService.get({ siteName }).then(convertFromBe),
+    async () => {
+      const siteSettings = await SettingsService.get({ siteName })
+      // TODO: reimplement actual functionality once netlify issue resolved
+      // const passwordSettings = await SettingsService.getPassword({ siteName })
```

Comment:
when should we re-enable this?

Reply:
There's an associated backend PR up (https://github.com/isomerpages/isomercms-backend/pull/828) - once it's approved we can re-enable this!

Reply:
Hey - after further discussion, we'll be keeping this feature hidden for now! The netlify change is in https://github.com/isomerpages/isomercms-backend/pull/828, the conditional has been changed in https://github.com/isomerpages/isomercms-frontend/pull/1344 (but is still disabled for all sites)